client/src/components/common/Button/ContainedButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  root: {
+    appearance: 'none',
+    background: 'hsl(22, 85%, 53%)',
+    borderRadius: '0.3rem',
+    border: 0,
+    color: 'white',
+    height: 48,
+    padding: '0 2rem',
+    minWidth: '8rem',
+    '&:hover': {
+      backgroundColor: 'hsl(22, 85%, 45%)',
+    },
+  },
+};
+
+function ContainedButton(props) {
+  const { classes, label } = props;
+  return <Button className={classes.root}>{label}</Button>;
+}
```

Comment:
1- Use Es6 when creating component, also we need only one file for component.
2- you can recieve props by this way .
3- please replace `label` by `children` to be more clear & expressive about prop's content 
4- Think what we can also have to pass to our button , as action for example ?!
```suggestion
const Button = ({ className, children,  ...props }) => {
```